docs/developers/linea-version/index.mdx

```diff
@@ -13,6 +13,34 @@ import TokenBridgeL2 from "!!raw-loader!/files/testnet/2023-06-13-release/TokenB
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+## Beta v1
+
+**Mainnet: TBC**
+
+**Linea Sepolia: September 23**
```

Comment:
It's not fully rolled out yet; let's say 26 for now (but I'm not sure it'll be rolled out completely today)